lib/FolderRow/styles.scss

```diff
@@ -23,7 +24,7 @@ $folder-row-spacing: $layout-spacing-base/2 !default;
   padding: $folder-row-spacing;
   background: $neutral-lightest;
   position: relative;
-  margin: 0 0 $layout-spacing-base/2;
+  margin: 6px 0 6px;
```

Comment:
🔴 can we move this to a var and then use it for the margin in item row? When we come to update it in the future it'll always catch us out when we update one and not the other.

Reply:
Love that idea